core/src/avm2/globals/Array.as

```diff
@@ -1,5 +1,207 @@
-// This is a stub - the actual class is defined in `array.rs`
 package {
+    [Ruffle(CallHandler)]
+    [Ruffle(InstanceAllocator)]
     public dynamic class Array {
+        public static const CASEINSENSITIVE:uint = 1;
+
+        public static const DESCENDING:uint = 2;
+
+        public static const UNIQUESORT:uint = 4;
+
+        public static const RETURNINDEXEDARRAY:uint = 8;
+
+        public static const NUMERIC:uint = 16;
+
+        // Some of these functions work on all Array-like objects in avmplus
+        // (i.e. Vector, ByteArray), but we only support calling them on Arrays
```

Comment:
as in, it's a bug? If so, can you make it clearer (`are supposed to work (...) but currently we only support`) and mark it as a TODO?